packages/edit-post/src/store/selectors.js

```diff
@@ -22,9 +22,9 @@ const EMPTY_ARRAY = [];
  *
  * @return {string} Editing mode.
  */
-export function getEditorMode( state ) {
-	return getPreference( state, 'editorMode', 'visual' );
-}
+export const getEditorMode = createRegistrySelector( ( select ) => () =>
+	select( preferencesStore ).get( 'core/edit-post', 'editorMode' ) ?? 'visual'
```

Comment:
Isn't the `??` unnecessary because we're using `setDefaults`?

Reply:
Yeah, it is. I added it back so that the unit tests are more logical. I suppose the unit tests could call `setDefaults` though. 🤔

Reply:
Nah all good that makes sense 👍